backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -75,7 +75,7 @@ public class Person extends OrganizationScopedEternalEntity implements PersonEnt
    * Note that for the purposes of all upserts, the <em>first</em> phone number in a
    * List<PhoneNumber> is considered to be the primary
    */
-  @OneToOne(fetch = FetchType.LAZY)
+  @OneToOne(fetch = FetchType.EAGER)
```

Comment:
Please read [other comment](https://github.com/CDCgov/prime-simplereport/pull/4243#discussion_r967240910) on my this was changed